Source/WebCore/accessibility/atspi/AccessibilityObjectAtspi.h

```diff
@@ -91,7 +91,7 @@ class AccessibilityObjectAtspi final : public RefCounted<AccessibilityObjectAtsp
     bool isDefunct() const;
     void stateChanged(const char*, bool);
     WEBCORE_EXPORT HashMap<String, String> attributes() const;
-    WEBCORE_EXPORT HashMap<uint32_t, Vector<RefPtr<AccessibilityObjectAtspi>>> relationMap() const;
+    WEBCORE_EXPORT HashMap<Atspi::Relation, Vector<RefPtr<AccessibilityObjectAtspi>>, IntHash<Atspi::Relation>, WTF::StrongEnumHashTraits<Atspi::Relation>>relationMap() const;
```

Comment:
OK, but don't write this mouthful more than once! Ideally you would use `using` for this. Something like:

`using RelationMap = HashMap<Atspi::Relation, Vector<RefPtr<AccessibilityObjectAtspi>>, IntHash<Atspi::Relation>, WTF::StrongEnumHashTraits<Atspi::Relation>>;`

would probably work? If you have any trouble with that, use a typedef.

(Also you're missing one space before `relationMap()`)